sources/documents/2_RTB/PianoDiProgetto/Analisi_rischi/Tipologia/Rischi_organizzativi/RO/RO2.tex

```diff
@@ -0,0 +1,21 @@
+\noindent\begin{minipage}{\textwidth}
+\subsubsection{RO2: Rischi relativi all'inesperienza}
```

Comment:
I titoli di RO2 e RO3 sono identici: dalla lettura credo sia da modificare quello di RO2 in "Rischi relativi alla collaborazione"